### ISSUE TITLE
JS directory aliases

### ISSUE THREAD

**Issue by kmeht** [bmun/huxley#509]
Currently, requiring modules in JS uses relative path, which sucks when we need to move modules around. Webpack offers a way to alias directories so we can use a full-path to them every time. For example, instead of:
```js
var Button = require('./Button');
```
we could do:
```js
var Button = require('components/Button');
```
where "components" is aliased to "absolute/path/to/static/js/huxley/components". Webpack has an option for these aliases. The only tricky thing would be getting it to work with Jest, but this article might be helpful: https://facebook.github.io/jest/docs/tutorial-webpack.html.

**Comment by kmeht** [comment]
Started in f0c0f59884b9d37b12157fd3205826577117b214.

**Comment by kmeht** [comment]
Resolved in #511.
